modules/runtime-config/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@storefront-x/runtime-config",
+  "version": "1.10.3",
```

Comment:
What logic is behind to call it 1.10.3? When it's a new thing? :)

Reply:
Storefront X uses same version for all of its packages.